src/coreclr/vm/methodtable.cpp

```diff
@@ -5188,7 +5188,7 @@ BOOL MethodTable::FindDispatchEntry(UINT32 typeID,
 
 #ifndef DACCESS_COMPILE
 
-void ThrowExceptionForAbstractOverride(
+void ThrowEntryPointNotFoundException(
```

Comment:
The contract for this method is wrong. It throws and allocates, so it cannot be LIMITED_METHOD_CONTRACT.

Reply:
Yep. I've a bit of clean-up to do in this PR before it is ready.